src/region.js

```diff
@@ -7,6 +7,8 @@
 // http://lostechies.com/derickbailey/2011/12/12/composite-js-apps-regions-and-region-managers/
 
 Marionette.Region = Marionette.Object.extend({
+  _cid: 'mnr',
```

Comment:
Looking again, I think `cidPrefix` is better because can be changed in the extensions of things. @paulfalgout I do not understand, why `_` on the end prefix instead `mnv` and `mncv`?